### PR TITLE
vscode: build before launching, launch from engine dir so that build artifacts can be found

### LINKED ISSUE #753: vscode support: automate running of build task before launching
https://github.com/OpenHV/OpenHV/pull/225

### PR DESCRIPTION
**Pull request by Phrohdoh**
like https://github.com/OpenHV/OpenHV/pull/225 but for RA2
closes #753

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+            "osx": {
+                "type": "mono",
+            },
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceRoot}/engine/OpenRA.Game.exe",
+            "cwd": "${workspaceRoot}/engine/",
+            "args": [
+                "Engine.LaunchPath=${workspaceRoot}",
+                "Engine.ModSearchPaths=${workspaceRoot}/mods",
```
Comment (abcdefg30) [inline .vscode/launch.json:19]: This is dropping a value.

**Review by abcdefg30 — COMMENTED** [review]
IIrc our other configurations use tabs for indentation, so we should do here too for consistency.

File: .vscode/launch.json (hunk 1 above)
Comment (Phrohdoh) [reply .vscode/launch.json:19]: I meant to comment on this when making the PR but forgot.
In my testing the now-dropped value wasn't needed.
Do we know why it was there previously?

Comment (abcdefg30) [reply .vscode/launch.json:19]: It is the mods directory of the engine directory. We probably need it for the modcontent mod.

Comment (Phrohdoh) [reply .vscode/launch.json]: > We probably need it for the modcontent mod.
Ah, you are correct.  I didn't think to test that.
I've added it back in (and switched indent to tabs).

**Review by Mailaender — APPROVED** [review]

**Review by abcdefg30 — APPROVED** [review]

**Pull request merged**